run_interest_forecasting_api_tests.sh

```diff
@@ -5,7 +5,7 @@ tags="not @smoke and not @ignore"
 if [ $# -gt 0 -a "$1" != "$environment" ];
 then
   environment="$1"
-  tags="not @ignore"
+  tags="@isoZapTest"
```

Comment:
Is this intended? IFS tests limited to Zap tests? What does isoZapTest mean?

Reply:
we don't need this tag now, is removed